Assignment5/question1.js

```diff
@@ -0,0 +1,20 @@
+/*
+    1. What is the difference between ++i and i++?
+        - ++i will increment value of i first and will then evaluate expression
+        - ++i will evaluate expression and will then increment the value of i
```

Comment:
you've written ++i at both places. Fix it

Reply:
done